src/applications/virtual-agent/components/webchat/WebChat.jsx

```diff
@@ -154,6 +154,22 @@ const WebChat = ({ token, WebChatFramework, apiSession }) => {
   );
 
   if (isRXSkill === 'true') {
+    // check if window.WebChat exists
+    if (window.WebChat) {
+      // find the send box element
+      const sendBox = document.querySelector(
```

Comment:
Is there any more robust way to find the input box? If msft changes the placeholder text this will stop working.

Reply:
No, there isn't, we found that MSFT doesn't support changing the text of the send box, there aren't any option for it in the styleOptions.
See here: https://stackoverflow.com/questions/59873091/not-able-to-customize-webchat-sendboxform-field